htdocs/product/price.php

```diff
@@ -378,6 +378,7 @@
 			$action = '';
 			setEventMessages($langs->trans("RecordSaved"), null, 'mesgs');
 			$db->commit();
+			$object->updatePriceFromExpression();
```

Comment:
This is outside the transaction ?  After the commit ? Why ?

Reply:
OK